### DISCUSSION TITLE
Forced password reset

### DISCUSSION THREAD

**Discussion by nickberthiaume** [community/community#17574] [Other Feature Feedback, Questions, & Ideas]
Seriously, who even thought that forcing someone to change their password is a good idea. I think a monkey could probably come up with a better feature than that. Whatever manager approved that should be fired. It's my password. I choose it. I don't mind if you *suggest* that I change it, but forcibly changing the password is probably the single dumbest thing I'd heard in the last decade.

**Comment by kocierik** [comment]
Why gitHub has forced you to change the password? It never happened to me...

**Comment by Rc-Cookie** [comment]
I totally agree. Suggesting is ok, but forcimg me to change it everywhere is so stupid. Like, there is no problem for GitHub at all if my account gets hacked. This is MY problem, MY decision.

Furthermore, how can GitHub know that my password is on a list of unsecure passwords? That means they have to store it unhashed and probably share it with some third party API, which is itself a real security threat, and really, shouldn't be legal. I suppose they state that somewhere in there terms of use, but that doesn't change the fact that it really shouldn't.

**Comment by KarimAkra** [comment]
![image](https://github.com/community/community/assets/144803230/66ecd266-bac4-4fdf-800a-38696d5be34b)
that's bullshit...
the heck am i supposed to do now i only memorize a single password for all my accounts...

**Comment by Zeroshiki-0** [comment]
Was just forced to change mine today. Super annoying, it's like they _want_ me to forget my password and never have access to this account again. I had it a specific way for a reason. Good thing I hardly ever use this, but pretty much guarantees that I'll forget what they forced me to change it to. They need to understand that we don't all have the memory of elephants. Passwords in general have become useless, anyway, with all the extra crap these companies force us to go through to login to _**our own accounts,**_ might as well let me keep it as what I chose.
